cves/CVE-2011-1817.yml

```
@@ -135,10 +171,13 @@ major_events:
 
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer: 
+  answer: |
+    During the fix, they also found out that there are some methods were called 
+    on a wrong thread, which certainly could make the browser stall or crashed.
```

```suggestion
    on a wrong thread, which certainly could make the browser stall or crash.
```
Another small grammar fix.